app/build.gradle.kts

```diff
@@ -54,11 +54,32 @@ android {
         release {
             isMinifyEnabled = true
             isShrinkResources = true
+
             proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
         }
+        debug {
+            applicationIdSuffix = ".debug"
+        }
+    }
+
+    productFlavors {
+        create("preprod") {
+            dimension = "env"
+
+            applicationIdSuffix = ".preprod"
+
+            buildConfigField("String", "BASE_URL", "\"https://$preprodHost\"")
+        }
+        create("prod") {
+            dimension = "env"
+
+            buildConfigField("String", "BASE_URL", "\"https://$prodHost\"")
+        }
```

Comment:
We should change the app icon background color for each of these 4 possibilities.